pcl_ros/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8)
 project(pcl_ros)
 
+add_compile_options(-std=c++11)
```

Comment:
Not sure this is necessary, since melodic targets bionic, where gcc defaults to 14 AFAIK

Will have to look and see how other ROS packages handle this